app/controllers/notifications_controller.rb

```diff
@@ -1,20 +1,25 @@
 class NotificationsController < ApplicationController
   before_filter :if_not_signed_in
-  before_action :set_notification, only: [:destroy]
```

Comment:
I like what @esBeee did to secure notifications https://github.com/julianguyen/ifme/pull/316/files#diff-e7d2878d7ccecd38f6e3a83b5cf85053R73